### DISCUSSION TITLE
Домашка после хакатона

### DISCUSSION THREAD

**Discussion by pingvin1308** [cleannetcode/CsharpForBeginers#23] [Новички]
Составить список пропущенных тем

https://metanit.com/sharp/tutorial/
https://docs.microsoft.com/ru-ru/dotnet/csharp/language-reference/language-specification/introduction

**Comment by emedvedeva** [comment]
Вот какие нерассмотренные темы я нашла:
Оператор yield
Partial классы
Индексаторы (https://docs.microsoft.com/ru-ru/dotnet/csharp/language-reference/language-specification/classes#indexers)
Многопоточность
Асинхронное программирование
Рефлексия
Dynamic Language Runtime
Сборка мусора
Классы ValidationResult, Validator и ValidationContext

**Comment by trulander** [comment]
Тема эвентов не раскрыта)), на самом деле мне кажется с эвентами мы как-то вскользь прошли тему и не успели пощупать, в отличие от делегатов.
